srcpkgs/xseticon/patches/fix-makefile.patch

```diff
@@ -0,0 +1,29 @@
+--- Makefile	2020-02-15 19:49:02.470295890 -0700
++++ Makefile	2020-02-18 16:38:26.171168225 -0700
+@@ -9,8 +9,8 @@
+ 
+ LIBS=${GLIB_LIBS} ${XLIB_LIBS} ${GD_LIBS}
+ 
+-PREFIX=/usr/local
+-BINDIR=$(PREFIX)/bin
++PREFIX=/usr
++INSTALL=install
+ 
+ all: xseticon
+ 
+@@ -20,11 +20,9 @@
+ xseticon: xseticon.o
+ 	gcc $^ ${LIBS} -o $@
```

Comment:
`s/gcc/\$(CC)/`

Reply:
Anyway, doesn't make have implicit rules for this one?
i.e.: executable from object files, and object files from source files.

It should be simple like:

```
LDFLAGS += $(LIBS)
```

else, it will discard our LDFLAGS